api/tests/core/users/test_api.py

```diff
@@ -2090,6 +2135,13 @@ def test_anonymize_beneficiary_users(self) -> None:
             for deposit in user_to_anonymize.deposits:
                 assert deposit.source == "Anonymized"
 
+            assert (
+                0
+                == users_models.GdprUserDataExtract.query.filter(
+                    users_models.GdprUserDataExtract.userId == user_to_anonymize.id
+                ).count()
```

Comment:
```suggestion
               users_models.GdprUserDataExtract.query.filter(
                    users_models.GdprUserDataExtract.userId == user_to_anonymize.id
                ).count() == 0
```
on met toujours le résultat attendu a droite pourquoi tou d'un coup tu veux le mettre a gauche ?